rllib/models/specs/specs_tf.py

```diff
@@ -8,7 +8,7 @@
 
 
 @DeveloperAPI
-class TFTensorSpecs(TensorSpec):
+class TfTensorSpec(TensorSpec):
```

Comment:
All other specs have the singular name: spec, NOT specs.
Also, we should - if possible - camelcase `Tf` to make it more readible and consistent (e.g. we also have `TfModel`, not `TFModel`).